File: URIPT1/puntos.cpp
#include <iostream>
#include <math.h>
using namespace std;


int main(int argc, char const *argv[])
{
	float x1,y1,x2,y2;
	cin>>x1>>y1;
	cin>>x2>>y2;

	x1 = x1 - x2;
	y1 = y1 - y2;
	float distancia;
	distancia = sqrt(pow(x1,2) + pow(y1,2));

	printf("%.4f\n", distancia);
	return 0;
}

File: URIPT1/puntos.py
import math


x1, y1 = [float(x1,y1) for x1 in raw_input().split()] 
x2, y2 =[float(x2,y2) for x2 in raw_input().split()]
val1 = x2 - x1
val2 = y2 -y1

Distancia = math.sqrt(pow(x2-x1, 2) + pow(y2-y1, 2))

print("{:.4f}".format(Distancia))
